API/NoDocs/Microsoft.Extensions.AmbientMetadata/ApplicationMetadataExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Assembly 'Microsoft.Extensions.AmbientMetadata.Application'
```

Comment:
Unclear on what this is.

Reply:
Metadata that gets used by enrichment. There are additional metadata components for environment-specific stuff, like Kubernetes or Service Fabric, they're not in this repo though.